01 - Community/People/Josh Plunkett.md

```diff
@@ -3,25 +3,60 @@ aliases:
 - Joshua Plunkett
 - Sklore
 - Sklex
-tags: 
-- 
+tags:
+- incubator
 publish: true
 ---
 
 # Josh Plunkett
 
-
+<!-- - GitHub: [Josh Plunkett](https://github.com/Josh Plunkett/) ^github-->
```

Comment:
What's the point of putting in a link that is commented out?

Reply:
Mmm, I think that is an accidental edit, from me "Moving it" down a line. That's not his username at all. I'll correct it in a commit.